src/main/resources/META-INF/jpa-schema-export.sql

```diff
@@ -1,1131 +0,0 @@
-
```

Comment:
@diptechexpo To prevent deletion of content in this `jpa-schema-export.sql` file, we need to also update the `CustomDispatcherServlet.java`.

I suggest changing from 
```java
entityScanner.findCandidateComponents("ai.elimu.model")
```
to
```java
entityScanner.findCandidateComponents(BaseEntity.class.getPackageName())
```

Since `BaseEntity.class.getPackageName()` will return "ai.elimu.entity."